website/content/docs/connect/config-entries/sameness-group.mdx

```diff
@@ -147,9 +146,23 @@ Specifies the local admin partition that the sameness group applies to. Refer to
 
 ### `DefaultForFailover`
 
-Determines whether the sameness group should be used to establish connections to services with the same name during failover scenarios. When this field is set to `true`, DNS queries and upstream requests automatically failover to services in the sameness group according to the order of the members in the `Members` list.
+Determines whether the sameness group should be used to establish connections to services with the same name during failover scenarios.
+
+When this field is set to `true`, DNS queries and upstream requests automatically failover to services in the sameness group according to the order of the members in the `Members` list. It impacts all services on the partition.
+
+When this field is set to `false`:
+  
+  - You must specify `destination_sameness_group` in order to configure an upstream. Refer to [Upstream configuration reference](/consul/docs/connect/proxies/proxy-config-reference#upstream-configuration-reference) for more information. 
```

Comment:
`destination_sameness_group` doesn't exist on a upstream definition from what I can tell. I know it was proposed at some point, but I don't believe it was ever implemented.